app/components/page-components/Chatbot/ChatbotTimers.vue

```diff
@@ -89,23 +96,35 @@ tbody tr {
   }
 }
 
+.chatbot-timers__timer-actions__container {
+  * > button {
```

Comment:
I'm not a CSS or LESS expert, but this strikes me as a weird rule to write.  Doesn't this just match any `button` that is a child of literally any other element?  If that's the case, isn't the `* >` completely unnecessary?  Maybe you can explain what this is matching.